### ISSUE TITLE
[BUG] "Invalid date" shown after added all entries

### ISSUE THREAD

**Issue by serrq** [davidhealey/waistline#698]
After the last update (f-droid) 3.6.0 Waistline always shows "invalid date" instead of time of entry.

![41691](https://user-images.githubusercontent.com/113377431/219938636-6c8db328-7533-4374-ac33-53857bbb9143.jpg)

**Comment by HelpfulCarrot** [comment]
For what it's worth, I'm also having this issue. I have been since F-Droid updated Waistline on my phone to version 3.6.0, on February 17th. I'd be happy to provide any logfiles, etc., upon request if that would be helpful.

**Comment by AlexeyGold2077** [comment]
I also have this issue. I have version 3.6.0 from F-droid. Can provide any logs.

**Comment by tanriol** [comment]
I also have this issue, and indeed it's wrong in the export
```json
        {
          "id": 107,
          "portion": "400",
          "quantity": "1",
          "type": "food",
          "dateTime": {},
          "category": "2"
        },
```

**Comment by EmilJunker** [comment]
That's weird. For me, exporting the database and then re-importing it fixes the problem (with only a few exceptions where it still shows "Invalid Date" afterwards)

**Comment by EmilJunker** [comment]
It is fixed in version 3.7.0. Alas, it seems like that version is not available on F-Droid or in the Play Store yet.

**Comment by davidhealey** [comment]
Ah weird, I'd pushed it to Google Play but apparently I had to send it to them for review, haven't had to do that before. Says it should be reviewed within 7 days. Not sure why f-droid hasn't pulled it yet...
